releasenotes/notes/0.6/qiskit-dependency-3f6b8d71cc4d2c31.yaml

```diff
@@ -5,11 +5,11 @@ upgrade:
     ``qiskit``. This change follows the move in upstream Qiskit to rename
     ``qiskit-terra`` to ``qiskit``. The minimum required version was increased
     from 0.24 for ``qiskit-terra`` to 0.44 for ``qiskit`` (equivalent to
-    `qiskit-terra`` 0.25). The maximum ``qiskit`` version was set to ``<1.0``
+    ``qiskit-terra`` 0.25). The maximum ``qiskit`` version was set to ``<1.0``
```

Comment:
I can't suggest a change to a large enough region, but the minimum version is now 0.45.3 and we no longer pin to less than 1.0, so all of that text can be removed.